client/my-sites/site-settings/jetpack-module-toggle.jsx

```diff
@@ -38,16 +42,32 @@ class JetpackModuleToggle extends Component {
 		isJetpackSite: PropTypes.bool,
 		activateModule: PropTypes.func,
 		deactivateModule: PropTypes.func,
+		path: PropTypes.string,
+		userID: PropTypes.number,
+		blogID: PropTypes.number,
 	};
 
 	handleChange = () => {
 		if ( ! this.props.checked ) {
+			this._recordTracksEvent( 'calypso_jetpack_module_toggle_activated' );
 			this.props.activateModule( this.props.siteId, this.props.moduleSlug );
 		} else {
+			this._recordTracksEvent( 'calypso_jetpack_module_toggle_deactivated' );
 			this.props.deactivateModule( this.props.siteId, this.props.moduleSlug );
 		}
 	};
 
+	_recordTracksEvent = name => {
+		const tracksProps = {
+			path: this.props.path,
```

Comment:
Do we really want to record the path with the site id (e.g. `/marketing/traffic/moderate-wasp.jurassic.ninja`), or should we sanitize it before (e.g. `/marketing/traffic/:site`)?

Reply:
I had considered that but didn't find a good library function for it. I also decided that sanitation isn't needed for privacy purposes, since this event already has `blogid`. But if you know of a good library function that does this then I will definitely add it!

Reply:
Do we even need this since Tracks has a built in attribute:  browserdocumentlocation

Reply:
I wasn't aware of that attribute, so we do not. I've removed it.